.github/workflows/build.yml

```diff
@@ -21,13 +21,11 @@ jobs:
       fail-fast: true
       matrix:
         python-version:
-          - "3.7"
           - "3.8"
           - "3.9"
           - "3.10"
           - "3.11"
           - "3.12"
-          - "pypy-3.7"
```

Comment:
Please add some newer PyPy version instead.  I don't want to claim PyPy compatibility if I don't run tests on PyPy in CI.